client/src/App.js

```diff
@@ -12,20 +12,34 @@ export default class App extends Component {
 		};
 	}
 
-	addToSavedList = movie => {
-		const savedList = this.state.savedList;
-		savedList.push(movie);
-		this.setState({ savedList });
-	};
+
+ addToSavedList = (movie, id) => {
```

Comment:
When you call `addToSavedList` in your `render()` method are you passing two parameters or just one? Are you using ID?